core/src/main/java/apoc/periodic/Periodic.java

```diff
@@ -64,6 +64,10 @@
         AtomicInteger failedBatches = new AtomicInteger();
         Map<String,Long> batchErrors = new ConcurrentHashMap<>();
 
+        String periodicId = UUID.randomUUID().toString();
+        if (log.isDebugEnabled()) {
+            log.debug("Starting periodic commit from `%s` in separate thread with id: `%s`", statement, periodicId);
```

Comment:
nit: might be worth removing the `if` and use the `{}` placeholder syntax of slf4j if applicable

Reply:
Removed `if` statements, with `{}` it doesn't seem to work with `neo4j.logging.Log`..

Reply:
If no access to that, might be better to keep those `if`s for consistency? I'm not sure it really affects performance too much, as it just created a few String instances here and there, but I haven't monitored that all

Reply:
You're right, better keep them. Restored